src/desktop_shortcuts/start_popup.bat

```diff
@@ -0,0 +1,24 @@
+echo off
+mode 70,19
+title Hello!
+cls
+echo.
+echo.
+echo                   .--,       .--,
+echo                  ( (  \.---./  ) )
+echo                   '.__/o   o\__.'
+echo                      {=  ^^  =}
+echo                       ^>  -  ^<
+echo        ___________.""`-------`"".____________
+echo       /  Hi, the GUI is starting!     O      \
+echo       \                      o               /
+echo       /  .    O       Please wait!       o   \
+echo       \                                      /         __
+echo       /        You can close this window!    \     _.-'  `.
+echo       \______________o__________o____________/ .-~^^        `~--'
+echo                     ___)( )(___        `-.___.'
```

Comment:
This looks awesome! How did you make it?

Reply:
I just googled "mouse ascii art" and modified one of the results